### ISSUE TITLE
Custom Catalogs: How can I point to a custom column?

### ISSUE THREAD

**Issue by Australian-Infomaniac** [calibre2opds/calibre2opds#19]
Sorry to put this here but I am working on this now and I am just unsure anyone is going to see my thread on the mobilereads forum

If anyone is in the mood to answer it, I'd really appreciate it!
https://www.mobileread.com/forums/showthread.php?p=4107148#post4107148

**Comment by Australian-Infomaniac** [comment]
I guess its kind of legit to post it here actually because something didn't work as expected

**Comment by calibre2opds** [comment]
i answered in the forum.

basically support for custom colums in search syntaxis a roadmap item that has not been implemented.   At the moment there is not much incentive to put in the effort to do it, although there is always a chance i decide to do it when I have nothng else on my plate as it is an interesting challenge.

**Comment by Australian-Infomaniac** [comment]
Aha ! Well that's a simple answer then! :)

It would add to the flexibility of the app immeasurably  - people could have anything on that front page :)

**Comment by itimpi** [comment]
I agree it would be a nice feature.   However it looks like quite a lot of work and Dave Pieron who developed the original support for the c2o search syntax is no longer active so I would have to work out how that part of the program works before even starting any enhancement.
